docker-compose.yml

```diff
@@ -3,18 +3,19 @@ services:
   bot1-pokego:
     build: .
     volumes:
-     - ./configs/config.json:/usr/src/app/configs/config.json
+     - ./configs:/usr/src/app/configs
+     - ./web:/usr/src/app/web
```

Comment:
This is needed because when building the image `bot1-pokego`, `/usr/src/app/web` is a volume and anything that is copied into there is ignored.

So we mount `./web` as a volume to include the `pogoweb` files in the container, and also include the json files added in there by the bot.